style-guide.md

```diff
@@ -0,0 +1,46 @@
+# Style guide
+This document is intended to synchronise styles across OccamResearchLtd outputs, including both R and Excel models.
+
+## Naming conventions
+Variables names should be snake case:
+
+Good:
+```
+starting_age
+```
+
+Bad:
+```
+startingAge
+starting.age
+Starting_Age
+```
```

Comment:
Should I change:
`anti.hypertensive.use `---> `use_antihypertensive`
`statin.use` ----> `use_statin`?

Also, what’s your view on hyphenated terms? For example, should we convert:
`anti-hypertensive `→ `antihypertensive`
`non-HDL-C `→ `nonHDLC`?
I’d suggest combining hyphenated terms into one word and using underscores only to separate main parts of the variable (like use_statin, prob_MI).

Alex, what do you think?